db/dbOps.js

```diff
@@ -58,22 +61,36 @@ async function createTables() {
 			ip INET
 		);
 	`);
+	await pool.query(`
+		CREATE TABLE countries(
+			country_code TEXT,
+			country_name TEXT
+		);
+	`);
 	await pool.end()
 }
 
 async function loadTables() {
-	for(let i=0; i<schools.length; i++) {
-		console.log(schools[i])
+	for (let i = 0; i < schools.length; i++) {
+		console.log(schools[i]);
 		const res = await pool.query(`
-			INSERT INTO locations(school_id, lat, lon, school) VALUES($1, $2, $3, $4) RETURNING *;`,
-			[schools[i].id, schools[i].lat, schools[i].lon, schools[i].school])
+			INSERT INTO locations(school_id, lat, lon, school, country_code) VALUES($1, $2, $3, $4, $5) RETURNING *;`,
+			[schools[i].id, schools[i].lat, schools[i].lon, schools[i].school, schools[i].country_code]);
 		console.log(res)
 	}
+
+	for (let i = 0; i < countryCodes.length; i++) {
```

Comment:
Same for whether this table is needed altogether, see previous comment on the table creation.